react/features/notifications/actions.js

```diff
@@ -78,19 +78,21 @@ export function showErrorNotification(props: Object) {
  * @param {Object} props - The props needed to show the notification component.
  * @param {number} timeout - How long the notification should display before
  * automatically being hidden.
+ * @param {number} uid - The unique ID of the notification. Only provided if we need to
```

Comment:
The commend says it's only provided under certain condditions, but the code sets a default.

Reply:
The default is necessary (it was there previously, I just moved it to a different location). The comment aims to say that we should only provide it when there is a real good cause to it (other than that, use the default)